src/ServerlessAutoSwagger.ts

```diff
@@ -151,9 +151,9 @@ class ServerlessAutoSwagger {
 
   predeploy = async () => {
     const stage = this.serverless.service.provider.stage;
-    const specifiedStages = this.serverless.service.custom?.autoswagger?.stages;
-    if(specifiedStages && specifiedStages.indexOf(stage) === -1) {
-      console.log(`Not deploying for stage: ${stage} due to custom.autoswagger.stages configuration.`);
+    const excludedStages = this.serverless.service.custom?.autoswagger?.excludeStages;
+    if(excludedStages && excludedStages.indexOf(stage) !== -1) {

```

Comment:
This can be simplified both for clarity and ease:
```suggestion
    if (excludedStages?.includes(stage)) {
```